package.json

```diff
@@ -35,7 +34,7 @@
     "@guardian/cdk": "48.5.1",
     "@guardian/eslint-config-typescript": "^1.0.11",
     "@guardian/prettier": "^2.0.0",
-    "@guardian/private-infrastructure-config": "git+ssh://git@github.com/guardian/private-infrastructure-config.git#v2.1.3",
+    "@guardian/private-infrastructure-config": "github:guardian/private-infrastructure-config#c6159b4a0a09d4f3a7a978a6fd48e37b1a2c8e3b",
```

Comment:
Does this need to be switched back to point at a branch now?

Reply:
not sure if @akash1810 made any progress on changing the release mechanism?